### PR TITLE
feat: update gas speed options

### PR DESCRIPTION
**Pull request by georgesmithishere**
https://www.notion.so/eb96722b397d42c9908002ca06af88e3?v=1f519ba1849347aa9740faedda4acf3b&p=4e6e0eecdda24ceb8eed1ca3c9f9897b

### CODE REVIEW

**Comment by r0x4xn** [comment]
App <strong>core-dapp</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://core-dapp-mg6yi9vyb-prepo.vercel.app'>https://core-dapp-mg6yi9vyb-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_JWVJ3WoV71UcOmB52qrcezjyBL3V"></hidden>

**Comment by r0x4xn** [comment]
App <strong>react-boilerplate</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://react-boilerplate-9aj65r9sl-prepo.vercel.app'>https://react-boilerplate-9aj65r9sl-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_3Lydru5gdN5RuqWFQBT8AZrbiROq"></hidden>

File: .gitignore [hunk 1]
```diff
@@ -88,9 +88,12 @@ artifacts*
 cache*
 cache
 
-# --------- prepo-constants ---------
+# --------- prepo-packages ---------
```
Comment (r0x4xn) [inline .gitignore:91]: nice!

**Review by r0x4xn — APPROVED** [review]
So nice to be able to test on core-dapp and react-boilerplate in 1 PR!

**Comment by georgesmithishere** [comment]
> So nice to be able to test on core-dapp and react-boilerplate in 1 PR!
yeah, thanks to you @r0x4xn , you did it!

**Pull request merged**